deployments/local/Dockerfile

```diff
@@ -1,11 +1,14 @@
 FROM gcr.io/google_appengine/nodejs
 
-RUN install_node v6.11.0
+RUN install_node v8.11.1
 
+WORKDIR /app/
 COPY . /app/
 
-RUN npm install --unsafe-perm
+RUN npm cache verify && find . -name node_modules -exec rm -Rf {} \; && find . -name package-lock.json -exec rm -Rf {} \;
```

Comment:
Can you add a comment for why/what you need the find things for. This seems a bit surprising to me.

Reply:
Turns out, it wasn't necessary.  Image build issues were actually caused by bad internet connection at office where I did the development.  Running the scripts at home makes it all just work.

I removed the unnecessary code.